### ISSUE TITLE
namebio.com: ads

### ISSUE THREAD

**Issue by Killstrike25** [uBlockOrigin/uAssets#14413]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I verified that the issue is not caused by removing any of the default lists
- [X] I did not enable external filter lists, or I verified that the issue still occurs without enabling external filter lists
- [X] I do not have custom filters/rules, or I verified that the issue still occurs without custom filters/rules
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625) (also mind browser built-in blockers)
- [ ] I did not answer truthfully to **all** the above checkpoints

### URL address of the web page

`https://namebio.com/`

### Category

ads

### Describe the issue

After using NameBio for a certain period of time it shows a Popup ad that forces you to watch for roughly 1 minute until you can close it. There are also Banner Ads displayed but they do not necessarily affect the normal use of their site. I tried to target the ad using the element zapper but it's very similar to ads on News sites and if you try to remove the ad it keeps the page locked so it ignore mouse clicks.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://user-images.githubusercontent.com/61766376/184877919-068ee960-b2c6-4269-b023-6606446c4884.png)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.43.0
Chromium: 104
filterset (summary): 
  network: 80385
  cosmetic: 45165
  scriptlet: 17529
  html: 0
listset (total-discarded, last updated): 
  default: 
    user-filters: 20-0, never
    easylist: 69422-51, 2d.7h.24m
    easyprivacy: 27334-127, 2d.7h.23m
    plowe-0: 3634-776, 4d.7h.27m
    ublock-abuse: 76-0, 12d.8h.29m
    ublock-badware: 4207-92, 2d.7h.22m
    ublock-filters: 32462-116, 2d.7h.21m
    ublock-privacy: 230-3, 12d.8h.28m
    ublock-unbreak: 1856-43, 2d.7h.20m
    urlhaus-1: 5212-0, 23h.18m
filterset (user): [array of 20 redacted]
trustedset: 
  added: [array of 7 redacted]
  removed: 
    chrome-scheme
switchRuleset: 
  added: [array of 1 redacted]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 428 ms (selfie)
  maxAssetCacheWait: 114 ms
popupPanel: 
  blocked: 0
```
</details>

**Comment by Procyon-b** [comment]
`ublock-filters: 32462-116, 2d.7h.21m`
You haven't updated your filters.

You can hide the auction banners with this filter:
```
namebio.com##.row.auction-widget
```

I didn't see a popup

**Comment by Killstrike25** [comment]
> `ublock-filters: 32462-116, 2d.7h.21m` You haven't updated your filters.
> 
> You can hide the auction banners with this filter:
> 
> ```
> namebio.com##.row.auction-widget
> ```
> 
> I didn't see a popup

I appreciate the very fast reply. My apologies I assumed this might be an issue long-term by adding a Custom Filter to the popup ad but I guess as long as the filter is enabled before seeing the element it can never force you to hit the "Close Button" on the Popup.

**WITH NO FILTERS**

How I managed to reproduce it just now was by pressing F5 repeatedly for approximately 10 refreshes then I saw the same popup I showed. By default, I do not think the popup is shown initially. 

**WITH FILTERS**

After adding these filters I no longer see any popups or banner ads thank you.

POPUP FILTER - UBLOCK Element Zapper

> ! 2022-08-16 https://namebio.com
> namebio.com##.modal-body
> namebio.com##.in.modal-backdrop

BANNER AD FILTER Thanks to [Procyon-b](https://github.com/Procyon-b)

> ! 2022-08-17 https://namebio.com
> namebio.com##.row.auction-widget
